src/lib/kadm5/admin.h

```diff
@@ -455,16 +467,11 @@ krb5_error_code kadm5_init_krb5_context (krb5_context *);
 
 krb5_error_code kadm5_init_iprop(void *server_handle, char **db_args);
 
-/*
- * kadm5_get_principal_keys is used only by kadmin.local to extract existing
- * keys from the database without changing them.  It should never be exposed
- * to the network protocol.
- */
 kadm5_ret_t    kadm5_get_principal_keys(void *server_handle,
                                         krb5_principal principal,
-                                        krb5_keyblock **keyblocks,
-                                        int *n_keys);
-
+                                        krb5_kvno kvno,
```

Comment:
Also, just in case, I believe MIT (and Heimdal) must commit -and implicitly have committed- to a stable API/ABI for kadm5.  (MIT and Heimdal may not have the same kadm5 _ABI_, but a subset of the kadm5 API is source compatible, and within each project, the ABI is mostly stable.)  Regardless of whatever level of API/ABI stability MIT is willing to accord to kadm5, minimizing gratuitous changes will be welcomed by projects like Russ' Wallet and Roland's krb5_admin.